modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration-suma.adoc

```diff
@@ -18,178 +19,32 @@ For more information about migrating traditional {productname} 4.3 clients to {s
-. After installation, ensure that the {micro} {microversion} Parent channel and Proxy child channels are added and synced from the menu:Admin[Setup Wizard -> Products] page.
-. In the {webui}, go to menu:Systems[Activation Keys] and create an activation key linked to the synced {micro} {microversion} channel.
-. Bootstrap your system as a minion using the menu:Systems[Bootstrapping] page.
-. Once the new machine is onboarded and displayed in the systems list, select the system and navigate to the menu:System Details[Install Package] page.
-. Install the packages [package]``mgrpxy`` and [package]``mgrpxy-bash-completion``.
-. Reboot the system.
-
-== Generate Proxy Config with spacecmd and Self-Signed Certificate
-
-.Task: Generate Proxy Config with spacecmd and Self-Signed Certificate
-You can generate a Proxy configuration using spacecmd.
-
-. SSH into your container host.
-
-. Execute the following command replacing the Server and Proxy FQDN:
-+
-----
-mgrctl exec -ti 'spacecmd proxy_container_config_generate_cert -- dev-pxy.example.com dev-srv.example.com 2048 email@example.com' -o /tmp/config.tar.gz
-----
-
-. Copy the generated config to the Proxy:
-+
-----
-mgrctl cp server:/tmp/config.tar.gz .
-----
-
-. Deploy the Proxy with:
-+ 
-
-----
-mgrpxy install podman config.tar.gz
-----
-
-== Generate Proxy Config with spacecmd and Custom Certificate
-
-You can generate a Proxy configuration using spacecmd for a custom certificates rather than the default self signed certificates.
-
-[NOTE]
+[IMPORTANT]
 ====
-2 GB represents the default proxy squid cache size. This will need to be adjusted for your environment.
+Before migrating the minions, ensure that the new proxy is already deployed and fully functional.
 ====
 
-.Task: Generate Proxy Config with spacecmd and Custom Certificate
-. SSH into your Server container host.
-. Execute the following command replacing the Server and Proxy FQDN:
-+
-
-----
-for f in ca.crt proxy.crt proxy.key; do
-  mgrctl cp $f server:/tmp/$f
-done
-mgrctl exec -ti 'spacecmd proxy_container_config -- -p 8022 pxy.example.com srv.example.com 2048 email@example.com /tmp/ca.crt /tmp/proxy.crt /tmp/proxy.key -o /tmp/config.tar.gz'
-----
-
-. Copy the generated config to the Proxy:
-+
-
-----
-mgrctl cp server:/tmp/config.tar.gz .
-----
-
-. Deploy the Proxy with:
-+ 
-
-----
-mgrpxy install podman config.tar.gz
-----
+.Task: Migrate minion between proxies
+. Log in to the {productname} Server {webui}.
+. Select menu:Systems[Systems List] from the left navigation.
+. Navegate to the Old 4.3 proxy page, and enter the tab menu "Proxy".
+. Select all system to "SSM".
+. Select menu:Systems[System Set Manager] from the left navigation.
+. Select the sub-menu "Misc" -> "Proxy"
+. From the drop down select the new proxy to migrate to.
+. Select "change proxy".
+. After this action minions will be migrated to the new proxy.
+. User can check the schedule progress. Verify if all sucessed
```

Comment:
. You can check the schedule progress to verify if all systems were successfully migrated.